css/main.css

```diff
@@ -27,9 +31,10 @@ header {
   color: white;
 }
 
-#desktop-nav, #desktop-nav ul {
+/* #desktop-nav,
+#desktop-nav ul {
   display: none;
-}
+} */
```

Comment:
- [ ] I would suggest removing unused/commented code for a better practice. This will keep your code clean and will be easier when you work in teams, this could help to avoid some issues later.